web/src/app/admin/AdminNumberLookup.tsx

```diff
@@ -50,24 +50,27 @@ const numInfoQuery = gql`
 export default function AdminNumberLookup(): JSX.Element {
   const [number, setNumber] = useState('')
   const [staleCarrier, setStaleCarrier] = useState(true)
-  const [lastError, setLastError] = useState(null as null | ApolloError)
 
-  const { data: numData } = useQuery(numInfoQuery, {
+  const [{ data: numData, error: queryError }] = useQuery({
+    query: numInfoQuery,
```

Comment:
Note that `pollInterval: 0,` has been removed. I did not see an equivalent for `urql`, though this may be fine to leave at the default.

Reply:
IIRC, it was there because, initially, carrier info was part of the query (and we'd get charged for each request). But I don't think it's necessary anymore since we broke that out into a mutation in the end.